SasquatchMac/SasquatchMacObjC/AppDelegate.m

```diff
@@ -22,11 +21,24 @@ - (instancetype)init {
   [MSMobileCenter start:@"8649b73e-6df0-4985-a039-8ab1453d44f3"
            withServices:@[ [MSAnalytics class], [MSCrashes class], [MSPush class] ]];
   [MobileCenterProvider shared].mobileCenter = [[MobileCenterDelegateObjC alloc] init];
-  return self;
+  [self updateMobileCenterState];
```

Comment:
This is definitely a timing issue between `applicationDidFinishLaunching` and `viewDidLoad`.
I really wonder why this doesn't happen in iOS apps. If this is just a macOS issue then we can probably document to start MobileCenter in `init` but we need to make sure this doesn't break anything because MobileCenter should start after application is ready for use.